README.md

```diff
@@ -87,6 +87,12 @@ Prerequisites:
    export REALM_CORE_DOWNLOAD_DIR=~/.realmCore
    ```
 
+   OS X users should also add following line to `~/.profile` in order for Android Studio to see this environment variable..
```

Comment:
I think this is only true if you use the default terminal right? I think @emanuelez and @beeender have been complaining about it not working.